ui/apps/platform/src/Containers/NetworkGraph/components/StyleGroup.tsx

```diff
@@ -71,7 +72,7 @@ const StyleGroup: React.FunctionComponent<StyleGroupProps> = ({
                 delete newData[key];
             }
         });
-        return newData;
+        return newData as CustomGroupNodeData;
```

Comment:
What do you think about `React.useMemo<CustomGroupNodeData>` instead?

https://www.freecodecamp.org/news/react-typescript-how-to-set-up-types-on-hooks/#set-types-on-usememo

Reply:
hm i'm not sure what the benefit of using `useMemo` is here since the function itself already uses the `useMemo` hook. maybe if we run into performance issues in the future we can look into this? i can add it as a comment for now so we don't forget

Reply:
Instead of `as` here in `return` statement how about `<CustomGroupNodeData>` useMemo above at line 68?